frontend/lib/src/util/createDownloadLinkElement.ts

```diff
@@ -32,7 +34,10 @@ const createDownloadLinkElement = ({
   } else {
     link.setAttribute("target", "_self")
   }
-  link.setAttribute("download", filename)
+
+  if (setDownloadAttribute) {
+    link.setAttribute("download", filename)
```

Comment:
The `download` attribute causes issues in conjunction with service workers, see [this 10 year old bug](https://issues.chromium.org/issues/40410035). When this attribute is set, a service worker does not seem to intercept the `fetch` requests corresponding to the link element click.
The filename passed from `Download` button has been empty (`""`) anyways and the browser seems to get the filename from the `Content-Disposition` header sent by the `/media` endpoint.

Reply:
Can you add this as a comment here:

> The download attribute causes issues in conjunction with service workers, see [this 10 year old bug](https://issues.chromium.org/issues/40410035). When this attribute is set, a service worker does not seem to intercept the fetch requests corresponding to the link element click.